data/blog/8th-wave-of-bitcoin-grants.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Eighth Wave of Bitcoin Grants"
+date: '2024-10-23'
+tags: ['OpenSats', 'grants', 'bitcoin', 'wave']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/59-bitcoin-8th-wave.jpg']
+summary: "OpenSats is funding 4 more open-source projects in the bitcoin ecosystem."
+---
+
+OpenSats is proud to announce its eighth wave of Bitcoin grants, dedicated to
+supporting innovative open-source projects that enhance the growth and
+resilience of the Bitcoin ecosystem.
+
+We remain committed to empowering developers worldwide who are building
+groundbreaking tools, improving user experience, and strengthening the overall
+security of Bitcoin and its users.
+
+This wave awards grants to four outstanding projects, each playing a vital role
+in advancing various aspects of the Bitcoin network. In no particular order, the
+recipients are:
+
+- [Citadel-Tech](#citadel-tech)
+
+- [Lampo](#lampo)
+
+- [Cashu Nutshell](#cashu-nutshell)
+
+- [PickhardtPayments Plugin](#pickhardtpayments-plugin)
+
+These grants are sourced from our General Fund, sustained by the generosity of
+donors who share our vision for a thriving, decentralized future. If you would
+like to support open-source projects that help Bitcoin flourish, please consider
+donating to the General Fund:
+
+<center>
+    <DonateToGeneralFundButton/>
+</center>
+
+Let's take a closer look at these projects to discover their purpose, impact,
+and the value they bring to the Bitcoin ecosystem..
```

Comment:
There are two periods at the end of this line. I recommend using either 1 period, or, if you are trying to write an ellipsis, use 3 periods.